templates/repository/conf/registry.xml.erb

```diff
@@ -1,155 +1,158 @@
-<?xml version="1.0" encoding="ISO-8859-1"?>
-
-<!--
-  ~ Copyright 2005-2011 WSO2, Inc. (http://wso2.com)
-  ~
-  ~ Licensed under the Apache License, Version 2.0 (the "License");
-  ~ you may not use this file except in compliance with the License.
-  ~ You may obtain a copy of the License at
-  ~
-  ~ http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing, software
-  ~ distributed under the License is distributed on an "AS IS" BASIS,
-  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  ~ See the License for the specific language governing permissions and
-  ~ limitations under the License.
-  -->
-<wso2registry>
-
-    <!--
-    For details on configuring different config & governance registries see;
-    http://wso2.org/library/tutorials/2010/04/sharing-registry-space-across-multiple-product-instances
-    -->
-
-    <currentDBConfig>wso2registry</currentDBConfig>
-    <readOnly>false</readOnly>
-    <enableCache>true</enableCache>
-    <registryRoot>/</registryRoot>
-
-    <dbConfig name="wso2registry">
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<!--
+  ~ Copyright 2005-2011 WSO2, Inc. (http://wso2.com)

```

Comment:
update license.

Reply:
Not needed since only some of the configs where updated.